src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,143 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private Object[] internalArray = new Object[10];
+    private int size = 0;
+
     @Override
     public void add(T value) {
-
+        if (size == internalArray.length) {
+            int newLength = (int) (internalArray.length * 1.5);
```

Comment:
[All magic numbers should be constants](https://mate-academy.github.io/jv-program-common-mistakes/java-core/collections/array-list.html#any-magic-numbers-should-be-constants) 
Fix it in all places